paddle/fluid/operators/fake_quantize_op.cc

```diff
@@ -217,12 +219,19 @@ class FakeQuantizeAbsMaxOpMaker : public framework::OpProtoAndCheckerMaker {
                                 bit_length));
         });
     AddComment(R"DOC(
-FakeQuantize operator
+This is a Base Op which support FakeQuantAbsMaxOpMaker and FakeQuantDequantAbsMaxOpMaker.
```

Comment:
Tiny issue:

"This is a Base Op which supports"
support -> supports

Same as following.

Reply:
Thx. Can I fix the syntax error in the next pr?